package.json

```diff
@@ -10,31 +10,31 @@
     "windows": "react-native run-windows"
   },
   "dependencies": {
-    "@react-native-community/checkbox": "^0.5.6",
+    "@react-native-community/checkbox": "^0.5.7",
     "@react-native-community/clipboard": "^1.5.1",
-    "@react-native-community/datetimepicker": "^3.0.8",
+    "@react-native-community/datetimepicker": "^3.3.0",
     "@react-native-community/masked-view": "^0.1.10",
-    "@react-native-community/slider": "4.0.0-rc.2",
-    "@react-native-picker/picker": "^1.9.4",
+    "@react-native-community/slider": "^4.0.0-rc.3",
+    "@react-native-picker/picker": "^1.9.12",
     "@react-navigation/bottom-tabs": "^5.8.0",
     "@react-navigation/drawer": "^5.9.0",
     "@react-navigation/native": "^5.7.3",
     "@react-navigation/stack": "^5.9.0",
     "@terrylinla/react-native-sketch-canvas": "creambyemute/react-native-sketch-canvas",
     "lowlight": "^1.17.0",
     "react": "16.13.1",
-    "react-native": "0.63.2",
+    "react-native": "^0.64.0-rc.1",
```

Comment:
Should this not have the "rc" on it?

Reply:
I updated the gallery using `npx react-native-windows-init --version 0.64.0 --overwrite` which automatically selects a copy of React Native to install for you. My guess is that it just selects the most up to date release that's compatible?